R/utilities.R

```diff
@@ -4406,9 +4362,9 @@ classify_fl <- function(
 
 }
 
-#' Classify DLBCLs according to genetic subgroups of Chapuy et al.
+#' Classify DLBCLs according to genetic subgroups.
 #'
-#' Use the feature weights from NMF model to assemble the binary matrix and classify DLBCL tumors based on C0-C5 system of Chapuy et al
+#' Use the feature weights from NMF model to assemble the binary matrix and classify DLBCL tumors
```

Comment:
Can you make this more detailed? What NMF model are these coming from? Is NMF really a model or do you mean from the NMF solution?

Reply:
I see that this was made less descriptive when you overloaded the function. I think you need to clarify what it does for each classifier now that this is generalized

Reply:
I have added more descriptions here. Also introduced an option for N1 group in Lacy classifier

Reply:
This is ready for re-review now!